### PR TITLE
fix(tools): load ts sources from core

### PR DESCRIPTION
**Pull request by bennypowers**
## What I did

1. more whoopsies from the dev server

### CODE REVIEW

File: tools/pfe-tools/dev-server/plugins/pfe-dev-server.ts [hunk 1]
```diff
@@ -130,6 +130,11 @@ function getRouter(options: PfeDevServerInternalConfig) {
       return next();
     })
 
+    // Redirect `core/pfe-core/controllers/thingy.js` to `core/pfe-core/controllers/thingy.ts` for requests not previously handled
```
Comment (zeroedin) [inline tools/pfe-tools/dev-server/plugins/pfe-dev-server.ts:133]: is there anything we should be worried about downstream here?  eg `lib/elements` in RHDS?

Comment (zeroedin) [reply tools/pfe-tools/dev-server/plugins/pfe-dev-server.ts:133]: follow up, should we allow for optional routes?

Comment (bennypowers) [reply tools/pfe-tools/dev-server/plugins/pfe-dev-server.ts:133]: downstream can do their own middleware i think

Comment (zeroedin) [reply tools/pfe-tools/dev-server/plugins/pfe-dev-server.ts:133]: You are correct and we are already doing as such:

```
export default pfeDevServerConfig({
  litcssOptions,
  tsconfig: 'tsconfig.json',
  middleware: [
    /** redirect requests for lightdom css to /elements */
    function(ctx, next) {
      const match = ctx.path.match(/^\/components\/(?<slug>[-\w]+)\/(?<path>.*)\.css$/);
      if (match) {
        const { slug, path } = /** @type{{ slug: string; path: string }} */ (match.groups);
        ctx.redirect(`/elements/rh-${slug}/${path}.css`);
      }
      return next();
    },
    /** redirect requests for /assets/* css to /docs/assets/prism.css */
    function(ctx, next) {
      if (ctx.path.startsWith('/assets/')) {
        ctx.redirect(`/docs${ctx.path}`);
      }
      return next();
    }
  ]
});
```

**Pull request merged**